spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
```

Comment:
- [ ] Here is the result I'm getting after setting up the the database, run the migrate and test:

![image](https://github.com/Habtamu210/Blog-App/assets/79058364/9fa51c03-d817-4545-9601-d2ef37d1f9ba)